CHANGELOG.md

```diff
@@ -24,6 +24,14 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.6.2...master
 
+### Removed
+- ReplicaConsistency#isConsistent(Supplier<Connection> replica)
+- Database#getId() returning String
+- Supplier<Connection> getConnectionSupplier();  -> getConnection DataSource
+### Added
+- ReplicaConsistency#isConsistent(Database replica)
```

Comment:
This API should allow us to verify replica consistency based on a cache, without polling a connection from the pool.